### PR TITLE
VEN-1023 | Improve loading time for single harbor pages

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Omit firstName & lastName from harbor page query
- Make BerthDetails fetch its data
- The customer name is fetched separately

### Closes :no_good_woman:
[VEN-1023](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1123): Improve loading time for single harbor pages

### Manual testing :construction_worker_man:
- Go to [Venepaikat page](https://venepaikka-admin.test.kuva.hel.ninja/harbors)
- Search for "Puotilan venesatama"
- Go to the corresponding page, the page should be loaded within 3-10 seconds
- In the piers table, when you expand the row it shows a loading spinner then it fetches the data

## Additional notes :spiral_notepad:
- The user is no longer able to sort the piers table by the customer names or search for a customer

### CODE REVIEW

File: src/common/berthDetails/BerthDetailsContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { useTranslation } from 'react-i18next';
+
+import BerthDetails from './BerthDetails';
+import LoadingSpinner from '../spinner/LoadingSpinner';
+import { BERTH_DETAILS_QUERY } from './queries';
```
Comment (TatuArvela) [inline src/common/berthDetails/BerthDetailsContainer.tsx:7]: This uses a query, so it should probably be in `features`

**Review by TatuArvela — APPROVED** [review]
One suggestion, but overall this LGTM 👍

File: src/common/berthDetails/BerthDetailsContainer.tsx (hunk 1 above)
Comment (TatuArvela) [reply src/common/berthDetails/BerthDetailsContainer.tsx:7]: `this` being `common/berthDetails`

Comment (Nizar-Rahme) [reply src/common/berthDetails/BerthDetailsContainer.tsx]: Moved `berthDetails` to `features` dir

**Pull request merged**